scripts/settings.js

```diff
@@ -403,6 +406,14 @@ export function toggleTestnet() {
     doms.domGuiBalanceStakingTicker.innerText = cChainParams.current.TICKER;
     doms.domPrefixNetwork.innerText =
         cChainParams.current.PUBKEY_PREFIX.join(' or ');
+
+    // Update testnet toggle in settings
+    if (cChainParams.current.isTestnet) {
+        doms.domTesnetToggler.checked = true;
+    } else {
+        doms.domTesnetToggler.checked = false;
+    }
```

Comment:
This code can be shortened to (keep in mind the `Tesnet` typo):
```suggestion
    doms.domTesnetToggler.checked = cChainParams.current.isTestnet;
```